src/docquery/document.py

```diff
@@ -104,10 +79,32 @@ def context(self) -> Tuple[(str, List[int])]:
     def preview(self) -> "Image":
         raise NotImplementedError
 
+    @staticmethod
+    def _normalize_box(box, width, height) -> List[int]:
+        return [
+            max(min(c, 1000), 0)
+            for c in [
+                int(1000 * (box[0] / width)),
+                int(1000 * (box[1] / height)),
+                int(1000 * (box[2] / width)),
+                int(1000 * (box[3] / height)),
+            ]
+        ]
+
+    @staticmethod
+    def _normalize_boxes(image, words, boxes) -> Tuple[List[str], List[List[int]]]:
```

Comment:
Can you add a comment and/or type signature clarifying what the expectations are for boxes here? E.g. what type are the boxes, and what scale are they? It seems like they're expected to be axis-aligned boxes (x1, y1, x2, y2) scaled to the height and width of the page?